code/game/machinery/computer/arcade.dm

```diff
@@ -591,81 +716,271 @@
+			else
+				eventdat += "<P ALIGN=Right><a href='byond://?src=\ref[src];wait=1'>Wait</a></P>"
+			eventdat += "<P ALIGN=Right><a href='byond://?src=\ref[src];close=1'>Close</a></P>"
+
+		if(ORION_TRAIL_BLACKHOLE)
+			eventdat += "You were swept away into the black hole."
+			eventdat += "<P ALIGN=Right><a href='byond://?src=\ref[src];holedeath=1'>Oh...</a></P>"
+			eventdat += "<P ALIGN=Right><a href='byond://?src=\ref[src];close=1'>Close</a></P>"
+			settlers = list()
+
+		if(ORION_TRAIL_LING)
+			eventdat += "Strange reports warn of changelings infiltrating crews on trips to Orion..."
+			if(settlers.len <= 1)
+				eventdat += "<br>Your crew's chance of reaching Orion is so slim the changelings likely avoided your ship..."
+				eventdat += "<P ALIGN=Right><a href='byond://?src=\ref[src];eventclose=1'>Continue</a></P>"
+				eventdat += "<P ALIGN=Right><a href='byond://?src=\ref[src];close=1'>Close</a></P>"
+				if(prob(10)) // "likely", I didn't say it was guaranteed!
+					lings_aboard = min(++lings_aboard,2)
+			else
+				if(lings_aboard) //less likely to stack lings
+					if(prob(20))
+						lings_aboard = min(++lings_aboard,2)
+				else if(prob(70))
+					lings_aboard = min(++lings_aboard,2)
+
+				eventdat += "<P ALIGN=Right><a href='byond://?src=\ref[src];killcrew=1'>Kill a crewmember</a></P>"
+				eventdat += "<P ALIGN=Right><a href='byond://?src=\ref[src];eventclose=1'>Risk it</a></P>"
+				eventdat += "<P ALIGN=Right><a href='byond://?src=\ref[src];close=1'>Close</a></P>"
+
+		if(ORION_TRAIL_LING_ATTACK)
+			if(lings_aboard <= 0) //shouldn't trigger, but hey.
+				eventdat += "Haha, fooled you, there are no changelings on board!"
+				eventdat += "<br>(You should report this to a coder :S)"
+			else
+				var/ling1 = remove_crewmember()
+				var/ling2 = ""
+				if(lings_aboard >= 2)
+					ling2 = remove_crewmember()
+
+				eventdat += "Oh no, some of your crew are Changelings!"
+				if(ling2)
+					eventdat += "<br>[ling1] and [ling2]'s arms twist and contort into grotesque blades!"
+				else
+					eventdat += "<br>[ling1]'s arm twists and contorts into a grotesque blade!"
+
+				var/chance2attack = alive*20
+				if(chance2attack)
+					var/chancetokill = 30*lings_aboard-(5*alive) //eg: 30*2-(10) = 50%, 2 lings, 2 crew is 50% chance
+					if(chancetokill)
+						var/deadguy = remove_crewmember()
+						eventdat += "<br>The Changeling[ling2 ? "s":""] run[ling2 ? "":"s"] up to [deadguy] and capitulates them!"
+					else
+						eventdat += "<br>You valiantly fight off the Changeling[ling2 ? "s":""]!"
+						eventdat += "<br>You cut the Changeling[ling2 ? "s":""] up into meat... Eww"
+						if(ling2)
+							food += 30
+							lings_aboard = max(0,lings_aboard-2)
+						else
+							food += 15
+							lings_aboard = max(0,--lings_aboard)
+				else
+					eventdat += "<br>The Changeling[ling2 ? "s":""] run away, What wimps!"

```

Comment:
So if there's 1 changeling, it says "The Changeling run away, What wimps!"

Even if you say this is a reference to changelings referring to themselves in the plural, it's still not very sound grammatically :P